### PR TITLE
Feat/multirage rebase

### PR DESCRIPTION
**Pull request by LaurensKubat**
The is a diff between this branch and the multirange meant to help reviewers of multirange see what changes were needed to rebase the multirange branch on main. THIS SHOULD NOT BE MERGED, AS THAT WOULD BE USELESS

### CODE REVIEW

File: smart-contracts/Cargo.toml [hunk 1]
```diff
@@ -14,9 +14,9 @@ members = [
   "contracts/cl-vault",
   "contracts/merkle-incentives",
   "contracts/range-middleware",
-  "contracts/dex-router-osmosis",
-  "contracts/lst-dex-adapter-osmosis",
-  "contracts/lst-adapter-osmosis",
+  "contracts/cw-vault-multi-standard",
```
Comment (lubkoll) [inline smart-contracts/Cargo.toml:17]: plz keep old contracts

File: smart-contracts/Cargo.toml [hunk 2]
```diff
@@ -39,11 +39,12 @@ dex-router-osmosis = { path = "contracts/dex-router-osmosis", features = ["libra
 lst-dex-adapter-osmosis = { path = "contracts/lst-dex-adapter-osmosis", features = ["library"] }
 lst-adapter-osmosis = { path = "contracts/lst-adapter-osmosis", features = ["library"] }
 
+
 # SDK
 cosmos-sdk-proto = {version = "0.21.1", default-features = false}
 
 # Osmosis
-osmosis-std = "0.25.0"
```
Comment (lubkoll) [inline smart-contracts/Cargo.toml:46]: plz keep version

**Comment by lubkoll** [comment]
looks like changes from main are overwritten at several places, plz double-check the conflict resolutions

**Review by lubkoll — CHANGES_REQUESTED** [review]

File: smart-contracts/Cargo.toml (hunk 2 above)
Comment (LaurensKubat) [reply smart-contracts/Cargo.toml:46]: Alright let me look at cw-roch versions, it still looks to be using v24 underlying causing type mismatches

Comment (lubkoll) [reply smart-contracts/Cargo.toml:46]: got it, let's see if that makes it into the abstract release, else we have to go with v0.24 longer

Comment (LaurensKubat) [reply smart-contracts/Cargo.toml:46]: After we talked to abstract and discussed on slack, decided to downgrade and upgrade as soon as new version is released

File: smart-contracts/Cargo.toml (hunk 1 above)
Comment (LaurensKubat) [reply smart-contracts/Cargo.toml:17]: error in the rebase of the toml, readded

**Pull request closed without merging**